Extension/lib/utils/element-collapser.js

```diff
@@ -275,7 +294,11 @@ var ElementCollapser = (function() { // jshint ignore:line
                 var srcSelector = createSelectorForSrcAttr(srcAttribute, tagName);
                 hideBySelectorAndTagName(srcSelector, tagName, shadowRoot);
             }
-            
+
+            // Remove important priority from element style
+            // https://github.com/AdguardTeam/AdguardBrowserExtension/issues/733
+            clearElStylesPriority(element, ['display', 'visibility', 'height', 'min-height']);
```

Comment:
We need this. Before the element is hidden with css-rule, which could be overwritten with element style with 'important' priority.